docs/meetings/2017/TechArea20171106.md

```diff
@@ -92,14 +92,16 @@ XRootd 4.7.1 has a bug that affects StashCache servers, but is not user-facing
 
 ### Last Week
 
--   GRACC changes; some services moving to the cloud
+-   GRACC changes
     - Changes to dashboard based on comments from e.g. LIGO and Minerva
+    - Overview of changes https://djw8605.github.io/2017/11/06/cleaning-up-gracc/
 -   StashCache changes
 
 
 ### This Week
 
-
+-   More GRACC changes
```

Comment:
Could you expand on this? Having missed the meeting I'm not sure what to expect. Is it the same stuff mentioned above in your blog post?

Reply:
Derek expanded on this offline. I'm fine with that.